c2rust-analyze/src/rewrite/expr/mir_op.rs

```diff
@@ -634,6 +644,13 @@ impl<'a, 'tcx> ExprRewriteVisitor<'a, 'tcx> {
                     self.emit(RewriteKind::UnsafeCastRawToRef { mutbl: true });
                     Some(Ownership::Mut)
                 }
+                Ownership::Cell if !early => {
+                    self.emit(RewriteKind::CastRawMutToCellPtr {
+                        ty: format!("{:?}", to.pointee_ty),
```

Comment:
Is `Debug` the right thing to use here?  I'm not sure whether we can rely on it producing well-formed Rust syntax.  I know there are some cases in `rewrite::ty` where we have to convert a `ty::Ty` to syntax - how is it handled there?

Reply:
I've matched what's in `rewrite::ty` with ca6bf033